__tests__/app.test.js

```diff
@@ -117,4 +117,59 @@ describe('GET: /api/topics', () => {
                 expect(body.msg).toBe('Path not found');
               });
           })
-    })
\ No newline at end of file
+    })
+
+    describe("GET /api/articles/:article_id/comments", () => {
+      it("200: responds with an array of comments for the given article_id", () => {
+        return request(app)
+          .get("/api/articles/5/comments")
+          .expect(200)
+          .then(({ body }) => {
+            const { comments } = body;
+            expect(comments).toEqual([{
```

Comment:
This test works only with this particular comment. Instead we want to make sure that all the comments (if many) have the right properties. Please refactor this to use a more flexible way and be able to check an array of multiple comments. (.toMatchObject() or .objectContaining() is recommended).

You will need some kind of loop for this, but don't forget to check the length of the array first.